test/org/apache/tomcat/util/http/TesterFastHttpDateFormat.java

```diff
@@ -0,0 +1,48 @@
+package org.apache.tomcat.util.http;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TesterFastHttpDateFormat {
```

Comment:
This needs to be called `TestFastHttpDateFormat` so it gets included in standard test runs.